tests/testProvenanceSetup.py

```diff
@@ -132,15 +136,15 @@ def testAddCmd(self):
         self.assertEquals(cmds[0][0], "recProv.py")
         self.assertEquals(cmds[0][1], "-v")
         self.assertEquals(cmds[0][2], "4")
-        
+
         self.setup.addWorkflowRecordCmd("dbingest.py", "-u ray".split(),
                                         "/usr/local/bin/dbingest.py")
```

Comment:
@srp3rd do you have an opinion about this? It looks suspicious to me (though if it is to be fixed, let's use a new ticket)